.gitignore

```diff
@@ -24,3 +24,9 @@
 database.db
 test.db
 *.iml
+
+# clojure-lsp
+.lsp/sqlite*.db
+
+# Calva VS Code extension
+.calva/output-window/output.calva-repl
```

Comment:
Should we ignore the entire `.lsp/*` and `.calva/*` folders or is it just these specific files that needs to be ignored? I'm not familiar with either of those so I felt like I should ask.

Reply:
At least for Calva I want to keep the option open to use that directory for config files, so it's better to be specific. I think it is the same for clojure-lsp. There I think it is already used as a place to put some config.